module/module.go

```diff
@@ -156,6 +164,7 @@ type Module struct {
 	logger                  logging.Logger
 	mu                      sync.Mutex
 	activeResourceStreams   map[resource.Name]peerResourceState
+	streamSourceByName      map[resource.Name]camera.VideoCodecStreamSource
```

Comment:
Q: why not embed `streamSource` in `peerResourceState`?

Reply:
Because `streamSourceByName` contains all cameras in the module. `activeResourceStreams` only contains the cameras which are currently streaming video.